example/package.json

```diff
@@ -9,6 +9,7 @@
     "preview": "vite preview"
   },
   "devDependencies": {
+    "algosdk": "^2.7.0",
```

Comment:
Can we remove this?

Reply:
Required for the very [basic example](https://github.com/algorandfoundation/liquid-auth-js/blob/develop/example/src/main.ts). We could just hardcode the bytes for the signing wallet

Reply:
We don't need the SDK. We can use any key pair to show how to use the SignalClient. 

Can we also move that "test" to a test suite instead of a runnable ".ts" file? :)

Reply:
resolved by: https://github.com/algorandfoundation/liquid-auth-js/pull/8